### PR TITLE
AppStudio in-context help - 1st draft

### PR DESCRIPTION
**Pull request by missmesss**
The 1st draft of the AppStudio-related definitions list (future in-context help).

@Hyperkid123 and @ryelo, I hope this new help topic is set up correctly. Are "tags" mandatory in metadata.yml? I wasn't sure which one to add but will do that if required. Thank you.

### CODE REVIEW

File: docs/help-topics/appstudio-incontext-help/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+kind: HelpTopic
+name: appstudio-incontext
+# tags:
```
Comment (Hyperkid123) [inline docs/help-topics/appstudio-incontext-help/metadata.yml:3]: Tags are not mandatory-you can remove the key. But add an empty line to the end of the file, please.

File: docs/help-topics/appstudio-incontext-help/appstudio-incontext.yml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  title: Git repository access authorization
+  content: |-
+    You can sign in to your private Git repository, a login window will open in a separate tab. Alternatively, you can use API token authentication. Maybe a list of supported Git hosting services?
+  links:
+    "[Link title]()"
+
+# Authorization section, option 2, Authorization
+- name: Authorize container registry
+  tags:
+  title: : Container registry authorization
```
Comment (Hyperkid123) [inline docs/help-topics/appstudio-incontext-help/appstudio-incontext.yml:40]: You have an extra `:`. That will break the DB seeding.

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

**Comment by Hyperkid123** [comment]
@missmesss Tags are not mandatory. But I would imagine would need some eventually. Usually, you want at least an `application` or `bundle` tag to associate the topic with some parts of the platform.

File: docs/help-topics/appstudio-incontext-help/appstudio-incontext.yml (hunk 2 above)
Comment (missmesss) [reply docs/help-topics/appstudio-incontext-help/appstudio-incontext.yml]: Good catch, thank you!

File: docs/help-topics/appstudio-incontext-help/metadata.yml (hunk 1 above)
Comment (missmesss) [reply docs/help-topics/appstudio-incontext-help/metadata.yml]: Will do, thank you. Didn't know .yml requires that too, we add empty lines in .adoc.

Comment (Hyperkid123) [reply docs/help-topics/appstudio-incontext-help/metadata.yml]: It's just a convention. Its not strictly "required"

**Review by Hyperkid123 — DISMISSED** [review]
@missmesss the format looks good. When would you like this change to be merged?

File: docs/help-topics/appstudio-incontext-help/appstudio-incontext.yml [hunk 3]
```diff
@@ -0,0 +1,167 @@
+# Names are UI elements names.
+# Tags to be kept empty for now. Tags will specify where in the app descriptions will be available.
+# Titles are "dictionary" articles titles.
+# Links to be external only. We don't know yet whether referencing to other terms within this list will be supported.
+
+# Terms that are 100% familiar to developers and probably shouldn't be part of this list: API token, Target port, Build command, Run command
+
+# DTUX-1136 Final design https://docs.google.com/document/d/1dMO7oCLdgayoNa3HvpNhvsGhBk9MoGG5TUnvbq8wDjs
+
+- name: Git repo URL or container image
```
Comment (Hyperkid123) [inline docs/help-topics/appstudio-incontext-help/appstudio-incontext.yml:10]: The topic name should only contain alphanumeric values. its basically an ID
```diff
-- name: Git repo URL or container image
+- name: git-repo-url-or-container-image
```

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**